test_scripts/TheSameApp/Policies/AppId/005_PTU_is_not_triggered_after_registration_app_with_different_app_id_during_timeout.lua

```diff
@@ -0,0 +1,90 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0204-same-app-from-multiple-devices.md
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3136
+--
+-- Description: Check that SDL does not trigger 2nd PTU for App2 from Mobile №2 after
+-- the first PTU for App1 from Mobile №1 is finished in case App1 and App2 are registered
+-- within 'ApplicationListUpdateTimeout'
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) Mobile №1 and №2 are connected to SDL and are consented
+-- 3) App1 is registered from Mobile №1 and triggers PTU
+--
+-- Steps:
+-- 1) App2 is registered from Mobile №2 during PTU for App1 from Mobile №1 is in progress
+--   Check: SDL does not send SDL.OnStatusUpdate and BC.PolicyUpdate to HMI during the app registration
+-- 2) First PTU is performed successful
+--   Check: SDL triggers new PTU after the first one is finished
+---------------------------------------------------------------------------------------------------
```

Comment:
SDL should not trigger new PTU in case second app was registered within 'ApplicationListUpdateTimeout

Reply:
@GetmanetsIrina Please find update in 0e0c08b